### PR TITLE
feat: APP-195 files preview

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-195?atlOrigin=eyJpIjoiYmIyYjMyYjRiMGEyNGJiNjllZjYyMGNkZGJiNDVmNDQiLCJwIjoiaiJ9

Also fixes https://regennetwork.atlassian.net/browse/APP-210

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

Check the files previews on:
https://deploy-preview-2403--regen-storybook.netlify.app/?path=/story/organisms-postfiles--public
https://deploy-preview-2403--regen-storybook.netlify.app/?path=/story/organisms-postfiles--private-locations

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@clevinson @erikalogie I still need to address the TODO mentioned in the PR description (pending some other work to be merged) but you can already follow the testing instructions to see the files preview from storybook

File: web-marketplace/src/components/atoms/index.ts [hunk 1]
```diff
@@ -11,8 +11,6 @@ export * from './OptimizedImage';
 export * from './PeerReviewed';
 export * from './PeerReviewed';
 export * from './ProcessStepCard';
-export * from './ProtectedRoute';
```
Comment (blushi) [inline web-marketplace/src/components/atoms/index.ts:14]: unrelated, just some clean up

**Comment by erikalogie** [comment]
This looks awesome

**Comment by blushi** [comment]
@erikalogie you can test the file previews on the data stream section as well now: https://deploy-preview-2403--regen-marketplace.netlify.app/project/test-project123

**Comment by erikalogie** [comment]
@blushi if there is only one file, then the icon isn't necessary

Test project | Regen Marketplace 2024-07-11 09-20-42

**Comment by blushi** [comment]
> @blushi if there is only one file, then the icon isn't necessary
> 
> Test project | Regen Marketplace 2024-07-11 09-20-42

@erikalogie fixed

**Comment by erikalogie** [comment]
Ok great, LGTM!

**Pull request merged**